www/front_src/src/Resources/Details/tabs/Details/DetailsCard/index.tsx

```diff
@@ -10,7 +10,7 @@ const useStyles = makeStyles((theme) => ({
     color: theme.palette.success.main,
   },
   container: {
-    height: 65,
+    height: 70,
```

Comment:
Do we still need this?

Reply:
No , we can come back with 65 for height , by using isCustomCard , issue is solved